### DISCUSSION TITLE
LOLIN I2C Motor library support request

### DISCUSSION THREAD

**Discussion by codystoica** [arendst/Tasmota#16890] [Ideas]
Hello
Please add support for latest D1 mini shield I2C dual motor driver shield based on HR8833.
Current support for address 0x30 is for WEMOS_MOTOR_V1 / V2 variants which are no longer manufactured.

https://github.com/wemos/LOLIN_I2C_MOTOR_Library

https://fr.aliexpress.com/item/1005003697301523.html?spm=a2g0o.order_list.0.0.21ef5e5bEnFCsf&gatewayAdapt=glo2fra

https://www.wemos.cc/en/latest/d1_mini_shield/hr8833_motor.html

**Comment by barbudor** [comment]
Don't dupplicate post
3 times the same request is a bit too much
